app/config/FrontendAppConfig.scala

```diff
@@ -81,7 +81,6 @@ class FrontendAppConfig @Inject() (val servicesConfig: ServicesConfig) {
     s"$accessibilityStatement?referrerUrl=${URLEncoder.encode(s"$platformHost${request.path}", "UTF-8")}"
 
   def languageTranslationEnabled: Boolean = servicesConfig.getBoolean("features.welshLanguage")
-  lazy val isExternalTest: Boolean        = servicesConfig.getBoolean("features.isExternalTest")
   lazy val noEmailPost: Boolean           = servicesConfig.getBoolean("features.noEmailPost")
 
```

Comment:
Please remove [here](https://github.com/hmrc/charities-registration-frontend/blob/main/app/config/FrontendAppConfig.scala#L93-L94) and its tests [here](https://github.com/hmrc/charities-registration-frontend/blob/main/test/config/FrontendAppConfigSpec.scala#L156-L162). Thanks